pkg/scheduler/util/topologies.go

```diff
@@ -38,6 +38,9 @@ func GetNamespacesFromPodAffinityTerm(pod *v1.Pod, podAffinityTerm *v1.PodAffini
 // PodMatchesTermsNamespaceAndSelector returns true if the given <pod>
 // matches the namespace and selector defined by <affinityPod>`s <term>.
 func PodMatchesTermsNamespaceAndSelector(pod *v1.Pod, namespaces sets.String, selector labels.Selector) bool {
+	if selector == nil {
```

Comment:
this check is ok, but also add a `continue` here so we don't call into this function knowingly with a bad selector:

https://github.com/kubernetes/kubernetes/blob/aaef1f46ef84a2d7bd71364165655e05219f2606/pkg/scheduler/internal/queue/scheduling_queue.go#L545-L548

Reply:
Sounds good! I'll update my pr

Reply:
I have update my PR, PTAL